modules/@apostrophecms/log/index.js

```diff
@@ -108,16 +108,41 @@ module.exports = {
     self.initFilters();
   },
   methods(self) {
+    // Keep those inside the methods because of performance.
+    // Do not move to the root because of tests.
     const isProduction = process.env.NODE_ENV === 'production';
+    const isTest = self.options.apos?.options?.test;
+    const utilInspect = require('node:util').inspect;
```

Comment:
TIL. I guess `util` is just an alias for `node:util` now. It goes back to 16, so that's fine.

Reply:
It's the recommended way by nodejs devs.